event_firstname/views/event_registration.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_event_registration_form_inherit_firstname">
+        <field name="name">event.registration.form</field>
+        <field name="model">event.registration</field>
+        <field name="inherit_id" ref="event.view_event_registration_form" />
+        <field name="name">event.registration.view.form.inherit.firstname</field>
+        <field name="arch" type="xml">
+
+            <field name="name" position="replace">
+
+                <field name="firstname"/>
+                <field name="lastname"/>
+            </field>
+        </field>
+    </record>
+
+
+    <record model="ir.ui.view" id="view_event_registration_tree_inherit_firstname">
+        <field name="name">event.registration.tree</field>
+        <field name="model">event.registration</field>
+        <field name="inherit_id" ref="event.view_event_registration_tree" />
+
+        <field name="arch" type="xml">
+            <field name="name" position="replace">
+                <field name="firstname"/>
+                <field name="lastname"/>
+            </field>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_registration_search_inherit_firstname">
+        <field name="name">event.registration.search</field>
+        <field name="model">event.registration</field>
+        <field name="inherit_id" ref="event.view_registration_search" />
+
```

Comment:
new fields should be searchable too

Reply:
@tschanzt reminder ;)

Reply:
to make fields searchable by default you must add `<fied name="fname" />` in the search view

Reply:
also, double check if `partner_id` or `name` is already here: must be searchable by fullname too

Reply:
`name` is available under `Participant`